src/cdk-experimental/scrolling/virtual-scroll-viewport.ts

```diff
@@ -102,8 +102,7 @@ export class CdkVirtualScrollViewport implements DoCheck, OnInit, OnDestroy {
   ngOnInit() {
     const viewportEl = this.elementRef.nativeElement;
     Promise.resolve().then(() => {
-      this._viewportSize = this.orientation === 'horizontal' ?
```

Comment:
nit: can remove `viewportEl` and use `this.elementRef.nativeElement` when listening below